recipes/edffile/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "5.0.0" %}
+
+package:
+    name: edffile
+    version: {{ version }}
+
+source:
+    path: src
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy  # does not compile against numpy's ABI. no 'x.x' needed
+
+    run:
+        - python
+        - numpy  # does not compile against numpy's ABI. no 'x.x' needed
+
+test:
+    # Python imports
+    imports:
+        - EdfFile
+
+about:
+    home: https://github.com/vasole/pymca/blob/master/PyMca5/PyMcaIO/EdfFile.py
```

Comment:
I would not say much more, but it is more :wink: 

The real question is: do we want sources bundled in the recipe? I am OK with lightweight cases like this, but I don't think we want that becoming common practice.

My Linux distro does mandate that we bundle the source together with the specfile and that leads to all sorts of trouble with people altering the source code directly and not documenting it properly.